### ISSUE TITLE
How to install the latest GoSublime?

### ISSUE THREAD

**Issue by dingrui37** [DisposaBoy/GoSublime#884]
Hi,all, I am want to use sublime text3 + gosublime for golang developing, but I have encontred some problems, but I have encountered some problems, can you give me help?

1）Install sublime text3 
2 ) Install package control according to  _https://packagecontrol.io/installation#st3_
3 ) Install gosublime, but it doesn't support now, so I follow the instructions below.

- Read the extension example and make sure you understand its contents. File an issue if you have any concerns.   //` I don't know what should I need to do?`

- Uninstall GoSublime //` Not installed before`

- Run the command git clone https://margo.sh/GoSublime to install GoSublime from the development branch. // `clone to C:\Users\xxx\AppData\Roaming\Sublime Text 3\Packages `

- Restart Sublime Text. The status bar should instruct you to press ctrl+.,ctrl+x or cmd+.,cmd+x which creates the margo extension file. Please read all comments in the file. //
``` there are some errors, 
margo.sublime build failed: press `ctrl +.`, `ctrl+x` to configure margo or check console for errors.
```

- If you've already switched to the new margo implementation, it might help to check the extension example for changes as there are some API breakages.

![sublim1](https://user-images.githubusercontent.com/22718365/48486612-00526a00-e857-11e8-817c-09053bd3e23a.jpg)

**Comment by DisposaBoy** [comment]
@dingrui37 what problems are you having? The instructions and the error message says to press <kbd>ctrl+.</kbd>, <kbd>ctrl+x</kbd>. Did you press that key binding?

**Comment by lifangmoler** [comment]
I am using Sublime build 3167, Can not find the `GoSublime` pacakge.

**Comment by DisposaBoy** [comment]
@lifangmoler Installation instructions are here https://github.com/DisposaBoy/GoSublime#installation--support

**Comment by dingrui37** [comment]
@DisposaBoy Code completion function can not be used.
![image](https://user-images.githubusercontent.com/22718365/48542251-a060cf80-e8f9-11e8-9164-bdc7180adf97.png)

**Comment by DisposaBoy** [comment]
@dingrui37 

* Did you read the extension example file https://margo.sh/s/extension-example ?

* The final step (and the error message in the status bar) says to press <kbd>ctrl+.</kbd>,<kbd>ctrl+x</kbd>. Did you press it?

**Comment by lifangmoler** [comment]
@DisposaBoy I followed the instruction. 
i did not see the final step and no status bar shown up after i restarted the Sublime.

**Comment by lifangmoler** [comment]
I do not have `GoSublime` installed ( it is not available to find via install package)
I cloned `GoSublime`  into this folder: ` /Applications/Sublime Text.app/Contents/MacOS/Packages` 
And then restarted the Sublime.  

Anything wrong?

**Comment by DisposaBoy** [comment]
@lifangmoler Package Control is not supported. You need to install GoSublime with git clone (as you did).

That path looks incorrect. The *editable* packages directory is somewhere inside your home directory like `/Users/<username>/Library...`. You can find it via the Sublime Text menu <kbd>Menu > View > Browse Packages</kbd>.

**Comment by DisposaBoy** [comment]
@lifangmoler If you're sure it's working, maybe you installed this branch some time in the past?

**Comment by lifangmoler** [comment]
`GoSublime` was working before.  I got the error `plugin_host has exited unexpectedly, plugin functionality won't be available until Sublime Text has been restarted`. Then I tried to install the latest version `Sublime` and found the `GoSublime` is not available to `Package Control` to install anymore. I googled the issues online and tried to figure it out by myself before leaving messages here. So I may have installed the branch yesterday
